docs/index.md

```diff
@@ -20,19 +20,19 @@ Post a question to Splunk Answers using the tag "Splunk Connect For Syslog"
 
 Join the #splunk-connect-for-syslog room in the splunk-usergroups Slack Workspace. If you don't yet have an account [sign up](https://docs.splunk.com/Documentation/Community/1.0/community/Chat)
 
-Please use the GitHub issue tracker to submit bugs or request enhancements: https://github.com/splunk/splunk-connect-for-syslog/issues
+Please use the GitHub issue tracker to submit bugs or request enhancements: [Issues](https://github.com/splunk/splunk-connect-for-syslog/issues)
 
 Get involved, try it out, ask questions, contribute filters, and make new friends!
 
 ## Contributing
 
-We welcome feedback and contributions from the community! Please see our [contribution guidelines](CONTRIBUTING.md) for more information on how to get involved.
+We welcome feedback and contributions from the community! Please see our [contribution guidelines](https://github.com/splunk/splunk-connect-for-syslog/blob/master/CONTRIBUTING.md) for more information on how to get involved.
 
 ## License
 
-* Configuration and documentation licensed subject to [CC0](LICENSE-CC0)
+* Configuration and documentation licensed subject to [CC0](https://github.com/splunk/splunk-connect-for-syslog/blob/master/LICENSE-CC0)
```

Comment:
same as above

Reply:
Yes, same as above, 404.

Reply:
same we should keep a copy of this file copied into the docs folder